packages/shared/components/TextHint.svelte

```diff
@@ -26,19 +26,19 @@
         } else if (secondary) {
             _classes = 'bg-gray-50 dark:bg-gray-800'
         } else if (success) {
-            _classes = 'bg-green-50 dark:bg-gray-800'
+            _classes = 'bg-green-50 dark:bg-green-50 border-green-500'
             _icon = 'checkmark'
-            _iconClasses = 'text-green-500 dark:text-red-500'
+            _iconClasses = 'text-green-500 dark:text-green-500'
         } else if (danger) {
-            _classes = 'bg-red-50 dark:bg-gray-800'
+            _classes = 'bg-red-50 dark:bg-red-50 border-red-500'
             _icon = 'success-error'
             _iconClasses = 'text-red-500 dark:text-red-500'
         } else if (warning) {
-            _classes = 'bg-yellow-50 dark:bg-gray-800'
+            _classes = 'bg-yellow-50 dark:bg-yellow-50 border-yellow-500'
             _icon = 'exclamation'
             _iconClasses = 'text-yellow-500 dark:text-yellow-500'
         } else if (info) {
-            _classes = 'bg-blue-50 dark:bg-gray-800'
+            _classes = 'bg-blue-50 dark:bg-blue-50 border-blue-500'
```

Comment:
in darkmode the texthints are a bit light, maybe something we could improve by adding bg-opacity in darkmode, but that would also require changing the text color. Just a note

Reply:
we are waiting on designs for the text hint this is okay for now